module-services/service-appmgr/include/service-appmgr/messages/SwitchBackRequest.hpp

```diff
@@ -11,12 +11,14 @@ namespace app::manager
 {
     class SwitchBackRequest : public BaseMessage
     {
+        std::unique_ptr<gui::SwitchData> data;
```

Comment:
Why have you moved one member here?

Reply:
The order of variables initialization must reflect the order of their declaration in the class, no instead of writing one more label (`private:`), I just moved it up :)

Reply:
@marziel  I agree with the order as you wrote. 
I would add one space line between `getData()` and `discardWhenAppNotOnStack` - it's just aesthetics, you can ignore it.

Reply:
For me, concisely written one after another looks good enough

Reply:
@marziel Is it consistent with our coding guideline and if there is none other parts of the code?